include/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics.hpp

```diff
@@ -61,21 +61,35 @@ class Dynamics
 
     virtual void print(std::ostream& anOutputStream, bool displayDecorator = true) const = 0;
 
-    /// @brief              Obtain dynamical equations function wrapper (pure virtual)
+    /// @brief              Get dynamical equations function wrapper (pure virtual)
     ///
     /// @return             std::function<void(const std::vector<double>&, std::vector<double>&, const double)>
 
+    static DynamicalEquationWrapper getDynamicalEquations(
```

Comment:
Because it's static, move down and capitalize method name?

Reply:
Oups! Yeah totally, that's a typo